APIHealthMonitorWeb/src/app/home/home.component.ts

```diff
@@ -1,19 +1,44 @@
 import { Component, OnInit } from '@angular/core';
 
 import { Title } from '@angular/platform-browser';
+import { Data } from '@angular/router';
+
+export interface DataElement {
+  name: string;
+  comments: string;
+}
+
+const DATA_ARRAY: DataElement[] = [
+  {name: 'Jhon', comments: 'Some comment 1'},
+  {name: 'Bob', comments: 'Some comment 2'},
+  {name: 'Kate', comments: 'Some comment 3'},
+  {name: 'Alice', comments: 'Some comment 4'},
+  {name: 'Piter', comments: 'Some comment 5'}
+];
```

Comment:
Please add already formated data to the objects with attr createdOn, like I showed above